tembo-operator/src/cloudnativepg/cnpg.rs

```diff
@@ -1055,6 +1059,20 @@ pub async fn reconcile_cnpg(cdb: &CoreDB, ctx: Arc<Context>) -> Result<(), Actio
             return Err(Action::requeue(Duration::from_secs(10)));
         }
     }
+    // Don't exec into the pod if it's not Ready yet
+    match is_pod_ready(&primary_pod_cnpg) {
+        Ok(true) => {
+            // Pod is ready, continuing
+        }
+        Ok(false) => {
+            info!("In reconcile_cnpg: Pod is not ready, requeuing 10 seconds");
+            return Err(Action::requeue(Duration::from_secs(10)));
```

Comment:
What do you think about changing this to 30 seconds?  I just don't want to hit the K8s API every 10 seconds waiting on a pod that will never come up.  (eg: Full PVC).

30 seconds isn't great either but I feel like we should quickly try again for pods that eventually will come up.